size-based.cabal

```diff
@@ -33,5 +33,5 @@ library
   build-depends:       base >=4.9 && <5,
                        dictionary-sharing >= 0.1 && < 1.0,
                        testing-type-modifiers >= 0.1 && < 1.0,
-                       template-haskell  >=2.5 && <2.23
+                       template-haskell  <2.24
```

Comment:
The old lower constraint was just wrong and I looked back throught Git history and could not find where is started being wrong.